### PR TITLE
Create template for PRs

### LINKED ISSUE #39: Create template s for PRs and Issues
## Depends on tasks:
none

## Description
Create the templates for PRs and issues in the repo to make workflow easier.
Templates should be created in the team's GitHub repository.

### User requirements:
- Discuss with the team and agree on templates for PR and issue.
- Add the  PR template to repo.
- Add the issue template to repo.

### Dev requirements and restrictions:
- investigate where the templates should be stored and how to create them. 

## UI Mocks
not required

## Other useful information
Maybe useful to check the [Git documentation](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/about-issue-and-pull-request-templates)

### PR DESCRIPTION
**Pull request by NadzeyaShpakouskaya**
## Summary
This PR is a part of the work to resolve #39.
Create a template for PRs to simplify and organize workflow. It also helps to improve the quality of code regarding the checklist in the PR template.

## UI Preview (Screenshots)
Screenshot 2023-05-18 at 13 54 06

## Changes
- Added `pull_request_template.md` file, which contains a template and checklist of action items before submitting new PR.

### CODE REVIEW

**Review by julyakmita — APPROVED** [review]
LGTM!
Nit: maybe we should add the example, the rule how to name a PR.

File: pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+<!-- Add a meaningful description of the changes you made -->
```
Comment (dersim-davaod) [inline pull_request_template.md:1]: Is that comment visible to the user who creates PR? Or only to those who edit the template?

What was your intention?

File: pull_request_template.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+<!-- Go through the checklist below to verify that your PR is good and ready for review -->
+
+## Checks to complete
+- [ ] I performed a self-review of my code.
+- [ ] PR title is in a conventional commit style.
+- [ ] My code follows the style guide of the project, I've built and run it, and no warnings or errors occurred.
+- [ ] I documented my code properly.
+- [ ] I added tests for my code.
+- [ ] Existing tests passed with my changes.
+- [ ] I set up PR settings properly (assignee, reviewers, link project, set milestone).
```
Comment (dersim-davaod) [inline pull_request_template.md:20]: nit:

```suggestion
- [ ] I've built and run my changes, and no warnings or errors occurred.
- [ ] All existing tests passed with my changes.
- [ ] My code follows our style guide.
- [ ] I've performed a self-review of my code.
- [ ] PR's title has a conventional commit style.
- [ ] Every non-private interface is documented properly.
- [ ] I've added tests for my code (if it makes sense).
- [ ] PR has assignee, reviewers, milestone; is linked to the project, and issue).
```

**Review by dersim-davaod — CHANGES_REQUESTED** [review]
Consider minor improvements:

**Review by iBYarchitect — APPROVED** [review]
LGTM

File: pull_request_template.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (NadzeyaShpakouskaya) [reply pull_request_template.md]: Fixed with https://github.com/ios-course/ironfoudation-team-project/pull/53/commits/c588e95bea3c7eea503564fd033ec5395fbc35a8

File: pull_request_template.md (hunk 2 above)
Comment (NadzeyaShpakouskaya) [reply pull_request_template.md]: Thanks for your advice, fixed with https://github.com/ios-course/ironfoudation-team-project/pull/53/commits/2c762c3cf6ae98cc13ed4b998f3e3cff9c42163c

**Review by dersim-davaod — APPROVED** [review]
LGTM

**Pull request merged**